src/main/java/ui/issuepanel/FilterPanel.java

```diff
@@ -61,7 +61,7 @@ public FilterPanel(UI ui, IModel model, PanelControl parentPanelControl, int pan
         
         getChildren().addAll(createPanelMenuBar(), createFilterBox());
         this.setOnMouseClicked(e-> {
-            ui.triggerEvent(new PanelClickedEvent(panelIndex));
+            ui.triggerEvent(new PanelClickedEvent(this.panelIndex));
```

Comment:
Nice catch! To prevent this from happening again we should factor the event handler out into its own method. That way it won't have a lexical reference to the local `panelIndex`.